packages/asset-registry/src/wrappedContracts/AssetConsumingDB.ts

```diff
@@ -4,24 +4,24 @@ import AssetConsumingDBJSON from '../../build/contracts/lightweight/AssetConsumi
 
 export class AssetConsumingDB extends GeneralFunctions {
     web3: Web3;
-    buildFile = AssetConsumingDBJSON;
 
     constructor(web3: Web3, address?: string) {
+        const buildFile: any = AssetConsumingDBJSON;
```

Comment:
This single change looks like a step backward.

Using :any here drops the types that should be imported from JSON (as far as I remember TS correctly resolved them when I was refactoring JSON's to reduce build size).

Reply:
If there's a problem with types maybe you can use:

`buildFile: typeof AssetConsumingDBJSON` or `buildFile: AssetConsumingDBJSON`

Reply:
All of those cause type issues. We could look for a better solution, but it's probably not worth the time/effort, assuming wrapped contracts will be removed/replaced soon enough.